atuin-server/src/database.rs

```diff
@@ -77,20 +78,23 @@ impl Postgres {
 
 #[async_trait]
 impl Database for Postgres {
+    #[instrument(skip_all)]
```

Comment:
The reasoning for `skip_all` is that it's usually better form to opt_in which fields you want to log (for PII as well as just for noisy structs like db handles).

In tracing 0.2, this will be the default: https://github.com/tokio-rs/tracing/issues/651